bin/pulumilocal

```diff
@@ -349,44 +341,62 @@ def get_service_endpoint():
-    endpoints = DEFAULT_CONFIG['config']['aws:endpoints']
-    if not endpoints:
-        entry = {service: endpoint for service in SERVICES}
-        endpoints.append(entry)
-
-    # write local config file
-    with open(file_name, 'w') as fp:
-        fp.write(yaml.dump(DEFAULT_CONFIG))
-    return file_name
-
-
-def prepare_cmd_args():
-    """
-    Constructs command line arguments for running the "pulumi" CLI
-    """
-    cmd_args = list(sys.argv)
-    if 'up' in cmd_args and '-s' not in cmd_args:
-        cmd_args += ['-s', PULUMI_STACK_NAME]
-    return cmd_args
+def set_localstack_pulumi_config(args: argparse.Namespace):
+    # LocalStack Endpoint
+    service_url = get_service_endpoint()
+    # Create argument list to pulumi config set-all
+    config_args = list()
+    config_args.append(PULUMI_CMD)
+    config_args.append("config")
+    if args.stack is not None:
+        config_args.append("--stack")
+        config_args.append(args.stack)
+    if args.cwd is not None:
+        config_args.append("--cwd")
+        config_args.append(args.cwd)
+    config_args.append("set-all")
+    config_args.append("--plaintext")
+    config_args.append("aws:region=us-east-1")
+    config_args.append("--plaintext")
+    config_args.append("aws:accessKey=test")
+    config_args.append("--plaintext")
+    config_args.append("aws:secretKey=test")
+    config_args.append("--plaintext")
+    config_args.append("aws:s3UsePathStyle=true")
+    config_args.append("--plaintext")
+    config_args.append("aws:skipCredentialsValidation=true")
+    config_args.append("--plaintext")
+    config_args.append("aws:skipRequestingAccountId=true")
+    for idx, service in enumerate(SERVICES):
+        config_args.append("--path")
+        config_args.append("--plaintext")
+        config_args.append(f"aws:endpoints[{idx}].{service}={service_url}")
+    # print(f"executing: {PULUMI_CMD} {config_args}")
+    process = subprocess.Popen(executable=PULUMI_CMD, args=config_args, env=os.environ, stdout=subprocess.PIPE)
+    process.wait()
+    # print(f"process.returncode: {process.returncode}")
 
 
 def main():
-    prepare_config_file()
-
-    if sys.argv[1] == 'init':
-        # TODO: proper CLI using click/argparse
-        cmd_args = [sys.argv[0], 'stack', 'init', PULUMI_STACK_NAME]
-    else:
-        cmd_args = prepare_cmd_args()
+    # Parse arguments from call to pulumi CLI that set the stack name and directory
+    parser = argparse.ArgumentParser(add_help=False)
+    parser.add_argument("command", help="pulumi primary command",
+                        type=str)
+    parser.add_argument("-s", "--stack", help="pulumi stack name",
+                        required=False,
+                        type=str)
+    parser.add_argument("-C", "--cwd", help="run in this directory",
+                        required=False,
+                        type=str)
+    args, extra = parser.parse_known_args()
+    print(f"command {args.command}")
```

Comment:
nit: should we remove this `print`? not sure it adds much value for the user to keep it..